### DISCUSSION TITLE
How to get cell x/y coordinates

### DISCUSSION THREAD

**Discussion by zhouyi0812** [flatironinstitute/CaImAn#1197] [Q&A] [answered]
Hello!

I wonder how can get the single x and y coordinates for each cell on CaIman? 

Here is my code for contour, but each cell has so many coordinates for it.
Thank you!

idx = cnm.estimates.idx_components[0]  
contour_dict = cnm.estimates.coordinates[idx]  
contour_coords = contour_dict['coordinates']  
contour_coords

**Comment by EricThomson** [comment, marked as answer]
What you are calling `contour_dict` is a coordinates dictionary that includes `CoM` key which is the center of mass. Generally you can do `contour_dict.keys()` to find what it includes.

Note these kinds of caiman usage questions are better suited for [GitHub Discussions](https://github.com/flatironinstitute/CaImAn/discussions). Issues are more for raising software problems/bugs/feature requests.  People will be more likely to search/find answers to these types of questions there moving forward.
